episodes/image-display.md

```diff
@@ -0,0 +1,358 @@
+){alt="Screenshot of searching 'matplotlib' on napari hub"}
+
+You should see 'napari Matplotlib' appear in the list (if not, try scrolling 
+further down the page). If we click on `napari matplotlib` this opens a summary 
+of the plugin with links to the documentation. There's also a useful 'Activity' 
+tab that summarises the number of installs and maintenance history of the plugin.
+
+Now that we've found the plugin we want to use, let's go ahead and install it 
+in Napari. Note that some plugins have special requirements for installation, 
+so it's always worth checking their napari hub page for any extra instructions. 
+In the top menu bar of Napari select:  
+`Plugins > Install/Uninstall Plugins...`
+
+![](fig/plugin-installation.png
+){alt="Screenshot of plugin installation window in Napari"}
+
+This should open a window summarising all installed plugins (at the top) and all 
+available plugins to install (at the bottom). If we search for 'matplotlib' in 
+the top searchbar, then 'napari-matplotlib' will appear under 'Available 
+Plugins'. Press the blue install button and wait for it to finish. You'll then 
+need to close and re-open Napari.
+
+If all worked as planned, you should see a new option in the top menubar under:  
+`Plugins > napari Matplotlib`
+
+::::::::::::::::::::::::::::::::::::: challenge 
+
+## Finding plugins
+
+Napari hub contains hundreds of plugins with varying quality, written by many 
+different developers. It can be difficult to choose which plugins to use!
+
+- Search for cell tracking plugins on [Napari hub](https://www.napari-hub.org/)
+- Look at some of the plugin summaries and 'Activity' tabs
+- What factors could help you decide if the plugin is well maintained?
+- What factors could help you decide if the plugin is popular with Napari users?
+
+:::::::::::::::::::::::: solution 
+
+## Solution
+
+### Is a plugin well maintained?
+
+Some factors to look for:
+
+**Last updated**  
+Check when the plugin was last updated - was it recently? This is shown in the 
+search list summary and under 'Maintenance' in the activity tab. In the activity 
+tab, you can also look at a graph summarising the 'commits' over the past year - 
+'commits' are made when someone updates their plugin with new changes.
+
+**Documentation**  
+Is the plugin summary (+ any linked documentation) detailed enough to explain 
+how to use the plugin?
+
+### Is a plugin popular?
+
+Some factors to look for:
+
+**Installs**  
+Check how many times a plugin has been installed - a higher number of installs 
+usually means it's more popular in the Napari community. The installs are shown 
+in the search list summary and under 'Usage' in the activity tab. In the 
+activity tab, you can also look at a graph summarising the installs over the 
+past year.
+
+**Image.sc**  
+It can also be useful to search the plugin's name on the 
+[image.sc](https://forum.image.sc/) forum to see if other people had good 
+experiences with it.
```

Comment:
```suggestion
experiences with it and/or the support provided by the community.
```

Reply:
By the community, do you mean the image.sc users or do you mean the developers who made it? Because if the former, then I think we should also mention the developers as well, as you could get support either from them or the image.sc users who have used it before.